app/src/layouts/appLayout/organizationsControl/organizationsPopover/organizationsItem/organizationsItem.jsx

```diff
@@ -87,7 +88,10 @@ export const OrganizationsItem = ({
   };
 
   const onClickCollapseButton = () => {
-    setIsCollapsed(!isCollapsed);
+    if (!isDisabled) {
+      setIsCollapsed(!isCollapsed);
+    }
```

Comment:
Can it be handled via `disabled` button attribute?
In that case no changes to the event listeners required.

Reply:
Done.